api/auth/v1/note.proto

```
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package noteV1;
```

Buf (1.55.1)

Files with package "noteV1" must be within a directory "noteV1" relative to root but were in directory "api/auth/v1".

(PACKAGE_DIRECTORY_MATCH)